src/bio.c

```diff
@@ -251,9 +251,11 @@ int wolfSSL_BIO_read(WOLFSSL_BIO* bio, void* buf, int len)
         }
     #endif
 
+    #ifdef USE_WOLFSSL_IO
```

Comment:
Should we maybe error out on a WOLFSSL_BIO_SOCKET BIO with WOLFSSL_USER_IO? Otherwise this may make it hard for users to investigate why their sockets aren't working.

Reply:
Agreed